.github/workflows/msbuild.yml

```diff
@@ -1,24 +1,50 @@
 name: Windows Build
 
 on:
-  push:
-    branches:
-      - master
-      - development
-
   workflow_dispatch:
```

Comment:
Not a big deal, but this is missing `# Triggers the workflow when manually dispatched` comment like the meson one has

Comments in the workflow_call section also don't quite align between the 2

Reply:
Resolved in 6999a7f7b526f4bacbe46686094e8c4221bb2655